pkg/upgrade/versions.go

```diff
@@ -19,10 +19,11 @@ var (
 	v1_12_0 = version{v: "1.12.0", upgrade: noop, next: &v1_13_0}
 	v1_13_0 = version{v: "1.13.0", upgrade: noop, next: &v1_13_1}
 	v1_13_1 = version{v: "1.13.1", upgrade: noop, next: &v1_14_0}
-	v1_14_0 = version{v: "1.14.0", upgrade: upgrade1_14_0, next: &v1_15_0}
-	v1_15_0 = version{v: "1.15.0", upgrade: upgrade1_15_0}
+	v1_14_0 = version{v: "1.14.0", upgrade: noop, next: &v1_15_0}
+	v1_15_0 = version{v: "1.15.0", upgrade: upgrade1_15_0, next: &v1_16_0}
+	v1_16_0 = version{v: "1.16.0", upgrade: noop}
```

Comment:
We *could* perhaps consider doing something about the Cassandra migration mentioned in Jaeger's changelog, but I think that this kind of migration isn't safe to be done without a downtime, and the operator can't decide what's a good time for a downtime...